### PR TITLE
Fix missed method on billingClient

### PR DESCRIPTION
**Pull request by salvobunetto**
Fix Missing class in some devices

### CODE REVIEW

**Comment by salvobunetto** [comment]
@googlebot I signed it!

File: GooglePlayPlugins/com.google.play.billing/Proguard/billing.txt [hunk 1]
```diff
@@ -8,7 +8,9 @@
 
 # Keep the AIDL interface
 -keep class com.android.vending.billing.**
+-keepclassmembers class com.android.vending.billing.** { *; }
```
Comment (Qing451800) [inline GooglePlayPlugins/com.google.play.billing/Proguard/billing.txt:11]: Remove this line and just add { *; } to previous line, does that work? Same comment for the other place.

**Review by Qing451800 — CHANGES_REQUESTED** [review]

File: GooglePlayPlugins/com.google.play.billing/Proguard/billing.txt (hunk 1 above)
Comment (salvobunetto) [reply GooglePlayPlugins/com.google.play.billing/Proguard/billing.txt:11]: > Remove this line and just add { *; } to previous line, does that work? Same comment for the other place.

I tried and it works!

**Comment by Qing451800** [comment]
Thanks for the PR. Please went ahead to use the plugin with your fix. We will take a process to merge your PR and republish the a new version of the plugin.

**Comment by salvobunetto** [comment]
> Thanks for the PR. Please went ahead to use the plugin with your fix. We will take a process to merge your PR and republish the a new version of the plugin.

Glad to have contributed, I published an update of my game with API 29, now everything is working properly

**Pull request merged**